src/MIDebugPackage/MIDebugPackagePackage.cs

```diff
@@ -306,9 +321,14 @@ private int MIDebugLog(uint nCmdLogOpt, IntPtr pvaIn, IntPtr pvaOut)
             return 0;
         }
 
-        private async void MIDebugExecAsync(string command)
+        private async Task MIDebugExecAsync(string command)
```

Comment:
>Task [](http://example.com/codeflow?start=22&length=4)

Is there a reason you did this instead of just throwing if called off of the UI thread?

Reply:
This was due to
```
##[warning]src\MIDebugPackage\MIDebugPackagePackage.cs(309,28): Warning VSTHRD100: Avoid "async void" methods, because any exceptions not handled by the method will crash the process.
C:\a\1\s\src\MIDebugPackage\MIDebugPackagePackage.cs(309,28): warning VSTHRD100: Avoid "async void" methods, because any exceptions not handled by the method will crash the process. [C:\a\1\s\src\MIDebugPackage\MIDebugPackage.csproj]

```

Reply:
Ah. I missed that this was async before.